src/migration/migrator.ts

```diff
@@ -513,7 +513,7 @@ export class Migrator {
       .orderBy(['timestamp', 'name'])
       .execute()
 
-    return executedMigrations.map((it) => it.name)
+    return executedMigrations.map((it) => it.name).sort()
   }
 
```

Comment:
Use the same sorting: first by timestamp, and only then by name. This is important for unordered migrations.

Reply:
The timestamp doesn't come from the file name. It's the execution time of the migration.